docs/cudf/source/conf.py

```diff
@@ -592,6 +592,8 @@ def on_missing_reference(app, env, node, contnode):
     ("py:obj", "cudf.Index.T"),
     ("py:obj", "cudf.Index.to_flat_index"),
     ("py:obj", "cudf.MultiIndex.to_flat_index"),
+    ("py:obj", "Series.pipe"),
+    ("py:obj", "DataFrame.pipe"),
```

Comment:
We should really figure out why some of these things show up in the nitpicky output eventually. I've forgotten why all of them end up here, but I'm sure we can do something better to fix at least some of them.

Reply:
Yeah definitely. I think some of these are still Sphinx bugs, but opened https://github.com/rapidsai/cudf/issues/18883 to revisit at some point